README.md

```diff
@@ -1,12 +1,12 @@
-Project Title: Music library 
+Project Title: Music library
```

Comment:
This really needs to be updated, it looks like the instructions from the track have been copy pasted in here. This is the place to include things like:
- A description of what the app actually does and what its features are
- installation instructions (especially regarding the database and the need to create environment variables),
- what technologies are required to use it, what the application dependencies are (`Express` and `MySQL2`), and what the development dependencies are (`nodemon`, `dotenv`, `mocha`, `chai`, `superTest`)
- information about the routes provided by the app, what type of HTTP verb they all require and an example request body would be a great addition,
- how to test the app,
- what you would do if you had more time (if applicable)

It would be worth describing how to spin up the database too, how did you get this running? Was it a docker container? If there are any commands needed to do this they should be listed here. Think about what you would like to see if you were coming to the project for the first time.

The formatting also needs looking at. You should always have a heading element (made with a single #) at the start of your readme containing your title and lay it out so it looks presentable. Have a look at [markdown live preview](https://markdownlivepreview.com/) so you can play with what the file will look like without having to do multiple commits to the repo to see the results, and have a look through this [guide to markdown](https://www.markdownguide.org/cheat-sheet/) too